framework/packages/abstract-std/src/objects/entry/contract_entry.rs

```diff
@@ -107,7 +107,7 @@ impl<'a> Prefixer<'a> for &ContractEntry {
 
 impl KeyDeserialize for &ContractEntry {
     type Output = ContractEntry;
-    const KEY_ELEMS: u16 = 1;
+    const KEY_ELEMS: u16 = 2;
```

Comment:
These changes are mandatory if you want to work with composite keys for these structures with custom cw-storage-plus impl